tests_practice/tasks_practice.py

```diff
@@ -0,0 +1,237 @@
+import string
+import math
+
+from typing import List
+
+
+def task_1(a: List, b: List):
```

Comment:
please name function as what they do. For example: task_1 will be task_1_get_common_elements()

Reply:
Done